.github/workflows/functional_test.yaml

```diff
@@ -0,0 +1,104 @@
+name: Functional Test Helm Chart
+
+on: pull_request
+
+jobs:
+  e2e-test:
+    name: Functional Test Helm Chart - ${{ matrix.container_runtime }}
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        container_runtime:
+          - "docker"
+          - "containerd"
+          - "cri-o"
+    env:
+      CI_SPLUNK_PORT: 8089
+      CI_SPLUNK_USERNAME: admin
+      CI_SPLUNK_HEC_TOKEN: a6b5e77f-d5f6-415a-bd43-930cecb12959
+      CI_SPLUNK_PASSWORD: helloworld
+      CI_INDEX_EVENTS: ci_events
+      CONTAINER_RUNTIME: ${{ matrix.container_runtime }}
+      KUBERNETES_VERSION: v1.21.2
```

Comment:
Should add the last couple of major k8s versions into the matrix?

Not needed in this PR, just as follow-up suggestion. Up to you

Reply:
Yeah, in future.